notebooks/metadata_extraction_with_llm_metadata_extractor.ipynb

```diff
@@ -0,0 +1,407 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0ac8d556-0ff2-4919-a159-2242a953ce44",
+   "metadata": {},
+   "source": [
+    "# LLMMetaDataExtractor: seamless metadata extraction from documents with just a prompt\n",
+    "\n",
+    "Notebook by [David S. Batista](https://www.davidsbatista.net/)\n",
```

Comment:
Others link github account or LinkedIn here, for example https://github.com/deepset-ai/haystack-cookbook/blob/bfa7b3abffcf02f149039863901998cfea0ed6a8/notebooks/extracting_metadata_filters_from_a_user_query.ipynb?short_path=ad0431f#L11 which I think makes more sense